cogs/audio.py

```diff
@@ -290,6 +290,9 @@ def _cache_max(self):
         setting_max = self.settings["MAX_CACHE"]
         return max([setting_max, self._cache_min()])  # enforcing hard limit
 
+    def _noppl_disconnect(self):
+        setting_nodis = self.settings["NOPPL_DIS"]
```

Comment:
Wouldn't the addition at line 2060 add it if not present? Just tried and it seemed to...